Proteomics/tutorials/WIP_proteinQuant_SIL.md

```diff
@@ -5,9 +5,16 @@ tutorial_name: proteinQuant_SIL
 ---
```

Comment:
The `topic_name` is proteomics and `tutorial_name` the same name as the filename